src/algebraic_geometry/structure_sheaf.lean

```diff
@@ -799,6 +799,26 @@ def basic_open_iso (f : R) : (structure_sheaf R).presheaf.obj (op (basic_open f)
   CommRing.of (localization.away f) :=
 (as_iso (show CommRing.of _ ⟶ _, from to_basic_open R f)).symm
 
+instance is_iso_to_global (R : CommRing) : is_iso (structure_sheaf.to_open R ⊤ : R ⟶ _) :=
+begin
+  let hom : CommRing.of _ ⟶ CommRing.of _ := algebra_map R (localization.away (1 : R)),
+  haveI : is_iso hom := is_iso.of_iso
+    ((is_localization.at_one R (localization.away (1 : R))).to_ring_equiv.to_CommRing_iso),
+
+  have : to_open R ⊤ = hom ≫ to_basic_open R (1 : R) ≫
```

Comment:
Could you extract this equality as a lemma and add an `@[elementwise]` attribute? It think that might be useful later. You might have to use some `@category_theory.category_struct.comp`-hackery so that lean accepts the `localization.at_one` ring morphism as a bundled morphism `⟶`. See for instance `to_open_comp_comap` at the bottom of this file.